code/modules/projectiles/guns/energy/special.dm

```diff
@@ -146,18 +146,16 @@
 	name = "super-kinetic accelerator"
 	desc = "An upgraded, superior version of the proto-kinetic accelerator."
 	icon_state = "kineticgun_u"
-	item_state = "kineticgun_u"
 	ammo_type = list(/obj/item/ammo_casing/energy/kinetic/super)
-	overheat_time = 14
+	overheat_time = 15
 	origin_tech = "combat=3;powerstorage=2"
 
 /obj/item/weapon/gun/energy/kinetic_accelerator/hyper
 	name = "hyper-kinetic accelerator"
 	desc = "An upgraded, even more superior version of the proto-kinetic accelerator."
 	icon_state = "kineticgun_h"
-	item_state = "kineticgun_h"
 	ammo_type = list(/obj/item/ammo_casing/energy/kinetic/hyper)
-	overheat_time = 12
+	overheat_time = 14

```

Comment:
I'd lower this to 13, personally, to keep the ratio the same as it was before--but that's me.